distributed/worker.py

```diff
@@ -2348,13 +2348,15 @@ async def execute(self, key: Key, *, stimulus_id: str) -> StateMachineEvent:
                     "Function:  %s\n"
                     "args:      %s\n"
                     "kwargs:    %s\n"
-                    "Exception: %r\n",
+                    "Exception: %r\n"
+                    "Traceback: %r\n",
                     key,
                     ts.state,
                     str(funcname(function))[:1000],
                     convert_args_to_str(args2, max_len=1000),
                     convert_kwargs_to_str(kwargs2, max_len=1000),
                     result["exception_text"],
+                    result["traceback_text"],
```

Comment:
Not meant as a blocking comment, but I'm wondering if this can get really long and we should maybe truncate.

Reply:
Logging tracebacks is standard behavior for logger.exception, logger.error and whenever exc_info=True is enabled. This can typically handled by loggers